src/test/java/com/squareup/kotlinpoet/FunSpecTest.kt

```diff
@@ -186,6 +186,23 @@ class FunSpecTest {
       |""".trimMargin())
   }
 
+  @Test fun functionParamWithKdocAndReturnKdoc() {
+    val funSpec = FunSpec.builder("foo")
+        .addParameter(ParameterSpec.builder("string", String::class.asTypeName())
```

Comment:
nit: `asTypeName()` is redundant, here and in the following lines, there are overloads that accept `KClass`.